velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -84,150 +84,226 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
   } else {
     VELOX_FAIL("Child Rel is expected in AggregateRel.");
   }
+  core::AggregationNode::Step aggStep;
+  // Get aggregation phase and check if there are input columns need to be
```

Comment:
This logic is non-intuitive. Would you describe it in more detail in the PR description?